.isort.cfg

```diff
@@ -0,0 +1,7 @@
+[settings]
+multi_line_output=3
+include_trailing_comma=True
+force_grid_wrap=0
+force_sort_within_sections=True
+use_parentheses=True
+line_length=80
```

Comment:
Small detail, but I think we could allow `100` , `80` is a bit extreme considering today's screen size

Reply:
Plus it would be the same as what is definer in `.pylintrc`